src/test/java/neoflix/_08_FavoriteFlagTest.java

```diff
@@ -19,7 +19,7 @@
     private static Driver driver;
 
     private static String userId;
-    private static final String email = "graphacademy.flag@neo4j.com";
+    private static final String email = "graphacademy.flag@neo4j.com"; // users get an error since this email already exists, might want to remove or change it before this module
```

Comment:
In the Node.js and Python courses I execute a `MERGE` statement to find or create a user with a specific email and ID before the tests are run rather than registering the user through the auth service.  

https://github.com/neo4j-graphacademy/app-nodejs/blob/main/test/challenges/08-favorite-flag.spec.js#L25-L28

Other than that, the user email should be isolated to this test alone so you could run the code to delete the user before the tests are run.  Or this could make line 28 redundant as you don't need to create the user and then merge the same user node.